server/graphql/v2/object/Host.js

```diff
@@ -378,7 +378,6 @@ export const GraphQLHost = new GraphQLObjectType({
                 required: true,
                 where: {
                   HostCollectiveId: host.id,
-                  ...(args.status === 'PENDING' && { approvedAt: null }),
```

Comment:
This is no longer needed since implementing the model check and fix in https://github.com/opencollective/opencollective-api/blob/main/checks/model/hosted-collectives.js#L45